### PR TITLE
feat: Add views for react rendered unsubscribe views

### PR DESCRIPTION
**Pull request by markstory**
I'm moving the unsubscribe views to react rendered to address silo boundary issues in hybrid cloud, remove more django views and help remove our reliance on the AuthMiddleware.

## Project unsubscribe
Screenshot 2023-11-01 at 4 46 59 PM

## Issue unsubscribe
Screenshot 2023-11-01 at 4 40 03 PM

The wording of the unsubscribe views is unchanged from before. The layout is different now as I felt the current pages were a bit too sparse.

Screenshot 2023-10-31 at 3 32 28 PM

### CODE REVIEW

File: src/sentry/api/endpoints/organization_unsubscribe.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ def get(self, request: Request, organization_slug: str, id: int, **kwargs) -> Re
         view_url = ""
         if hasattr(instance, "get_absolute_url"):
             view_url = str(instance.get_absolute_url())
+
+        display_name = ""
+        user = request.user
+        if hasattr(user, "get_display_name"):
+            display_name = str(user.get_display_name())
```
Comment (mrduncan) [inline src/sentry/api/endpoints/organization_unsubscribe.py:60]: Is re-setting this here intentional? It looks like it's already set above on line 50

**Review by mrduncan — APPROVED** [review]
Much nicer layouts!

File: src/sentry/api/endpoints/organization_unsubscribe.py (hunk 1 above)
Comment (markstory) [reply src/sentry/api/endpoints/organization_unsubscribe.py:60]: Ugh, git rebase took some creative license that I missed.

**Review by GabeVillalobos — APPROVED** [review]

File: src/sentry/api/endpoints/organization_unsubscribe.py (hunk 1 above)
Comment (vbro) [reply src/sentry/api/endpoints/organization_unsubscribe.py:56]: LGTM
Curious question: are there reasonable defaults for `view_url` and `display_name`, or is empty string okay?

**Review by vbro — APPROVED** [review]

File: src/sentry/api/endpoints/organization_unsubscribe.py (hunk 1 above)
Comment (markstory) [reply src/sentry/api/endpoints/organization_unsubscribe.py:56]: In practice the default values are never used. I couldn't figure out how to appease mypy typing with anything other than a runtime check though.

**Pull request merged**